assets/scss/components/_card.scss

```diff
@@ -0,0 +1,158 @@
+.resource-block .resource-block-left-large {
+  margin-top: 70px;
+}
+
+.resource-block-left-stripes {
+  display: block;
+  width: 8.75rem;
+  height: 8.75rem;
+  margin-top: 7.5rem;
+  margin-left: -3.75rem;
+  background: repeating-linear-gradient(
```

Comment:
@louisefindlay23 would you mind refactoring all of this CSS into SCSS in order to maintain a bit more consistency with the rest of the code base?

Reply:
Sure, @hborrelli1. Thanks for the feedback. 

Have grouped the code using inheritance and merged duplicate `@media` rules. Have altered the `z-index` and `position` of the sidebar which should improve the responsiveness on smaller screens.